src/mahoji/commands/buy.ts

```diff
@@ -82,6 +88,37 @@ export const buyCommand: OSBMahojiCommand = {
 				content: `Removed ${cost} from your bank.`
 			};
 		}
+		if (stringMatches(name, 'fossil island notes')) {
+			const cost = new Bank().add('Numulite', 300);
+			if (user.minionIsBusy) {
+				return 'Your minion is busy.';
+			}
+			if (user.QP < 3) {
```

Comment:
Do we want to take into account the `quantity` option? I dont know how many commands it will take to get them all, but if its a lot, people might want it.

Reply:
It's 10 items total, but I can add in for quantity to do this